source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -173,92 +189,125 @@ that matches your query, use the filtered positional operator
-- ``garlic``: the value to push onto the ``toppings`` array
-
-Next, add the matching criteria in your ``arrayFilters`` object in your update
-operation's ``options`` parameter. This object is an array of queries
-that specify which array elements to include in the update. To add the
-"garlic" topping to order items of type "pizza" and "large size", pass the
-following ``arrayFilters``:
-
-.. code-block:: javascript
-
-   arrayFilters: [
-     { orderItem.type: "pizza" },
-     { orderItem.size: "large" }
-   ]
-
-The following snippet shows the complete update method:
-
-.. literalinclude:: /code-snippets/crud/arrayFilters.js
-   :language: javascript
-   :start-after: start arrayFiltersOne example
-   :end-before: end arrayFiltersOne example
-
-After you run the method, your customer document for Steve
-Lobsters resembles the following:
+This example shows how to update certain matching array elements in the
+following sample documents, which describe shopping lists made for specific
+recipes:
 
 .. code-block:: javascript
    :copyable: false
 
    {
-     name: "Steve Lobsters",
-     address: "731 Yexington Avenue",
+     _id: ...,
+     date: "11/12/2023",
+     items: [
+       { item: "Scallions", quantity: 3, recipe: "Fried rice" },
+       { item: "Mangos", quantity: 4, recipe: "Salsa" },
+       { item: "Pork shoulder", quantity: 1, recipe: "Fried rice" },
+       { item: "Sesame oil", quantity: 1, recipe: "Fried rice" }
+     ]
+   },
+   {
+     _id: ...,
+     date: "11/20/2023",
      items: [
-       { type: "pizza", size: "large", toppings: ["pepperoni", "garlic"] },
-       { type: "pizza", size: "large", toppings: ["pineapple", "ham", "garlic"], ...},
-       ...
+       { item: "Coffee beans", quantity: 1, recipe: "Coffee" }
      ]
    }
 
-Example
-```````
+Suppose you want to increase the quantity of items you purchase for a
+recipe on your ``"11/12/2023"`` grocery trip. You want to double the quantity if
+the item meets the following criteria:
 
-Suppose Steve Lobsters wants to adjust their order to add "salami" as a
-topping to all pizzas that have pepperoni. To perform the update, use
-the filtered positional operator as follows:
+- The item is for the ``"Fried rice"`` recipe.
+- The item name does not include the word ``"oil"``.
+
+To double the ``quantity`` field in the matching array
+entries, use the filtered positional operator as shown in the following
+code:
 
 .. literalinclude:: /code-snippets/crud/arrayFilters.js
    :language: javascript
-   :start-after: start arrayFiltersTwo example
-   :end-before: end arrayFiltersTwo example
+   :start-after: start arrayFiltersIdentifier example
+   :end-before: end arrayFiltersIdentifier example
+   :emphasize-lines: 3, 6-11
+   :dedent:
 
-After you run the update method, your customer document for Steve
-Lobsters resembles the following:
+After you run the update operation, the documents resemble the
+following. The update multiplied the ``quantity`` field by ``2`` for
```

Comment:
Suggestion:

I think the wording "the documents resemble the following." works only with a ":" because it presents a list. I think changing this paragraph into a list of changes (and non-changes), and therefore modifying the wording of the intro sentence could be helpful:

```suggestion
This update operation performs the following changes as shown in the output:
```
(followed by a list of each update)